src/jinja2/utils.py

```diff
@@ -837,7 +950,9 @@ class Markup(markupsafe.Markup):
     def __new__(cls, base, encoding=None, errors="strict"):  # type: ignore
         warnings.warn(
             "'jinja2.Markup' is deprecated and will be removed in Jinja"
-            " 3.1. Import 'markupsafe.Markup' instead.",
+            " 3.1. Use Environment.get_markup_class and "
```

Comment:
What to do with this warning? Simply remove it, as it is supposed to be removed in 3.1 anyway?